test/e2e/suites/gc_managed/gc_managed_test.go

```diff
@@ -159,6 +159,127 @@ var _ = ginkgo.Describe("[managed] [gc] EKS Cluster external resource GC tests",
 		Expect(err).NotTo(HaveOccurred())
 		Expect(arns).To(BeEmpty(), "there are %d service load balancers (elb) still", len(arns))
 	})
+
+	ginkgo.It("[managed] [gc] should cleanup a cluster that has ELB/NLB load balancers using AlternativeGCStrategy", func() {
```

Comment:
As ginkgo can run tests out of order, this may cause an issue. The new `It` may run first and so the deployment is modified to use the alternate GC strategy and then the original/existing "It` is run against the alternative GC strategy, so we end up not testing the original strategy.

A couple of ways around this:

1. Move the new tests  into the existing It
2. Change the `Describe` to be **Ordered**